### PR TITLE
fix: fix nav-menu content scroll issue and fix nav-bar to top of the screen on mobile

### PR DESCRIPTION
**Pull request by SamanthaAO**
# Description
- add styling to fix nav-bar to the top of screen on mobile
- fix issue for when the mobile menu is open being able to scroll down to the content and move past the menu

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-website/src/app/css/style.css [hunk 1]
```diff
@@ -327,15 +332,14 @@ fast-badge::part(control) {
   site-navigation .anchor-wrapper {
     display: flex;
     flex-direction: column;
-    position: absolute;
+    position: fixed;
```
Comment (radium-v) [inline sites/fast-website/src/app/css/style.css:335]: fixed-position elements nested in fixed-position elements are redundant

File: sites/fast-website/src/app/css/style.css [hunk 2]
```diff
@@ -348,12 +352,12 @@ fast-badge::part(control) {
...
   }
 
-  .site-navigation.debounce .anchor-wrapper {
+  site-navigation.debounce .anchor-wrapper {
     transition-duration: 0s;
   }
 
   site-navigation.opened .anchor-wrapper {
-    transform: translateY(0);
+    transform: translateY(-100%);
```
Comment (radium-v) [inline sites/fast-website/src/app/css/style.css:360]: This is causing the nav to slide up from the bottom of the screen for me

File: sites/fast-website/src/app/css/style.css (hunk 1 above)
Comment (SamanthaAO) [reply sites/fast-website/src/app/css/style.css]: This has been resolved per your help through discord :)

File: sites/fast-website/src/app/css/style.css (hunk 2 above)
Comment (SamanthaAO) [reply sites/fast-website/src/app/css/style.css]: This has been resolved per your help through discord :)

File: sites/fast-website/src/app/css/style.css [hunk 3]
```diff
@@ -330,11 +337,20 @@ fast-badge::part(control) {
     align-items: center;
   }
 
-  .header .site-navigation {
+  .header .site-navigation{
```
Comment (radium-v) [inline sites/fast-website/src/app/css/style.css:340]: space

Comment (SamanthaAO) [reply sites/fast-website/src/app/css/style.css]: oops. Nice catch. Thanks.

**Pull request merged**